### ISSUE TITLE
Trying phel\router

### ISSUE THREAD

**Issue by Chemaclass** [mabasic/mariobasic-n7#13]
I see you implemented your own [router logic](https://github.com/mabasic/mariobasic-n7/blob/main/src/router.phel), when we didn't have anything similar in phel. 

Maybe you would like to try out the new phel router component? https://github.com/phel-lang/router

I create this issue just as an ideas to let you know in case you want to try this out 😄

**Comment by mabasic** [comment]
Yes, thank you for reminding me. I was planning to switch to phel/router, but since the site was finished at the time, I started working on other projects.

In the future I will probably switch to phel/router since it seems better than what I have conjured 😊.

**Comment by Chemaclass** [comment]
I created a new repository https://github.com/phel-lang/web-skeleton in which I use the new `phel/router`, so you can use it as a template or example whenever you decide to use the router. Also, feel free to suggest any improvements or ideas in that new repo, as always!
